pkg/topology/topology.go

```diff
@@ -52,11 +52,14 @@ type Hint struct {
 type Hints map[string]Hint
 
 func getDevicesFromVirtual(realDevPath string) (devs []string, err error) {
-	if !filepath.HasPrefix(realDevPath, "/sys/devices/virtual") {
-		return nil, fmt.Errorf("%s is not a virtual device", realDevPath)
+	relPath, err := filepath.Rel("/sys/devices/virtual", realDevPath)
```

Comment:
@mythi I'm guessing you didn't see where filepath.Rel is called. It's here. Sorry if my guess is incorrect.